SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -342,4 +344,11 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	// before the actor holding the position for all the hierarchy, it can immediately attempt to migrate back.
 	// Using this timestamp, we can back off attempting migrations for a while.
 	uint64 AuthorityReceivedTimestamp;
+
+	// Contains the frame at which the actor (or its owner) was seen outside its worker's authoritative region.
+	// Used to delay migration by one frame, to allow Pre-Replication events to take place in the regular replication loop.
```

Comment:
Can we be more explicit here?

I would prefer this to highlight that:
- Contains the frame at which the Actor hierarchy's was marked to be migrated to another worker.
- Used to delay migration  of all Actors in the hierarchy by one frame. This ensures PreReplication is called for each hierarchy Actor, which would not happen if ReplicateActor is called on the current frame.